lyncs_quda/clover_field.py

```diff
@@ -34,89 +34,109 @@ class CloverField(LatticeField):
      * Only rho is mutable.  To change other params, a new instance should be created
      * QUDA convention for clover field := 1+i ( kappa csw )/4 sigma_mu,nu F_mu,nu (<-sigma_mu,nu: spinor tensor)
      *  so that sigma_mu,nu = i[g_mu, g_nu], F_mu,nu = (Q_mu,nu - Q_nu,mu)/8 (1/2 is missing from sigma_mu,nu)
+     * Apparently, an input to QUDA clover object, coeff = kappa*csw 
+     *  wihout a normalization factor of 1/4 or 1/32 (suggested in interface_quda.cpp) 
     """
 
+    _children = {}
```

Comment:
I understand why you redefine _children, but I don't like it... I would maybe store the children using as key `(cls, parent)`. What do you think?